OnBoarding/Configuration.php

```diff
@@ -75,30 +81,30 @@ public function getConfiguration()
                                     'savepoint',
                                 ),
                                 'page' => array(
-                                    'index.php/product/new',
-                                    'index.php/product/form',
+                                    $this->generateSfUrl($router, 'admin_product_new'),
+                                    $this->generateSfBaseUrl($router, 'admin_product_form', array('id' => static::FAKE_ID)),
                                 ),
                                 'selector' => '#form_step1_name_1',
                                 'position' => 'right',
                             ),
                             array(
                                 'type' => 'tooltip',
                                 'text' => $this->translator->trans('Fill out the essential details in this tab. The other tabs are for more advanced information.', array(), 'Modules.Welcome.Admin'),
-                                'page' => 'index.php/product/form',
+                                'page' => $this->generateSfBaseUrl($router, 'admin_product_form', array('id' => static::FAKE_ID)),
                                 'selector' => '#tab_step1',
                                 'position' => 'right',
                             ),
                             array(
                                 'type' => 'tooltip',
                                 'text' => $this->translator->trans('Add one or more pictures so your product looks tempting!', array(), 'Modules.Welcome.Admin'),
-                                'page' => 'index.php/product/form',
+                                'page' => $this->generateSfBaseUrl($router, 'admin_product_form', array('id' => static::FAKE_ID)),
                                 'selector' => '#product-images-dropzone',
                                 'position' => 'right',
                             ),
                             array(
                                 'type' => 'tooltip',
                                 'text' => $this->translator->trans('How much do you want to sell it for?', array(), 'Modules.Welcome.Admin'),
-                                'page' => 'index.php/product/form',
+                                'page' => $this->generateSfBaseUrl($router, 'admin_product_form', array('id' => static::FAKE_ID)),
```

Comment:
Instead of generating the URL and manipulating strings (4 times for this one), can you please group your calls in variables?